baseledger/x/bridge/keeper/attestation_handler.go

```diff
@@ -248,6 +248,16 @@ func (a AttestationHandler) Handle(ctx sdk.Context, att types.Attestation, claim
 				)
 				return sdkerrors.Wrap(err, "could not undelegate from validator specified on claim")
 			}
+
+			// this is a specific case of validator missbehaving when we want to slash all tokens from validator
```

Comment:
comment seems misleading: if we undelegate everything for that validator, we also remove the minimum stake amount that cannot be controlled through the deposit contract. It does not have to do anything with misbehaving validator?